docker/chronicle.yaml

```diff
@@ -105,10 +107,12 @@ services:
       - PGPASSWORD=chronicle
     image: &chronicleImage chronicle-amd64:${ISOLATION_ID:-local}
     entrypoint: /bin/bash
+    links:
+      - "opa-init:opa-init"
     command:
       - -exc
       - |
-        sleep 10
+        sleep 120
```

Comment:
Maybe needn't be this long once we get things working?

Reply:
There's hopefully a better way to coordinate - it's quite fiddly but yes..

Reply:
We are using stop-on-container-exit to return test results, but we have an ephemeral init process, lots of fun

Reply:
May be worth a followup ticket.